go/oasis-node/cmd/node/node.go

```diff
@@ -129,6 +130,7 @@ type Node struct {
 	TransactionSchedulerWorker *txnscheduler.Worker
 	MergeWorker                *merge.Worker
 	SentryWorker               *workerSentry.Worker
+	SentryGrpcWorker           *workerGrpcSentry.Worker
```

Comment:
Hm could this be part of the regular sentry worker instead of having a separate one for gRPC? And then maybe have configuration flags like `sentry.consensus.enabled` and `sentry.grpc.enabled` for specific things?

Reply:
yeah makes sense, will do